docs/source/install/kernel_install.rst

```diff
@@ -53,3 +53,23 @@ For example, using conda environments:
 The ``--name`` value is used by Jupyter internally. These commands will overwrite
 any existing kernel with the same name. ``--display-name`` is what you see in
 the notebook menus.
+
+Using virtualenv or conda envs, you can make your IPython kernel in one env available to Jupyter in a different env. To do so, run ipykernel install from the kernel's env, with --prefix pointing to the Jupyter env:
```

Comment:
It may be helpful to add a sentence on "why" you would want to use a kernel from another environment.